src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -1570,8 +1605,15 @@ bool DoorLockServer::getMaxNumberOfCredentials(chip::EndpointId endpointId, Cred
         status = emberAfPluginDoorLockGetNumberOfFaceCredentialsSupported(endpointId, maxNumberOfCredentials);
         break;
     case CredentialTypeEnum::kAliroCredentialIssuerKey:
+        status = emberAfPluginDoorLockGetNumberOfAliroCredentialIssuerKeyCredentialsSupported(endpointId, maxNumberOfCredentials);
+        break;
     case CredentialTypeEnum::kAliroEvictableEndpointKey:
+        status = emberAfPluginDoorLockGetNumberOfAliroEvictableEndpointKeyCredentialsSupported(endpointId, maxNumberOfCredentials);
```

Comment:
This part doesn't seem right.  There are no separate caps for evictable and non-evictable endpoint keys, right?  Just a single cap across both types.

This needs some careful checking.

Reply:
"There are no separate caps for evictable and non-evictable endpoint keys, right?" - Agreed. Isn't this what the door lock should handle and return the values correctly in the callback. Are you saying we should use the max number of endpoint keys and do something with that?